### PR TITLE
Put JWT in brkt_config, not as file

### PR DESCRIPTION
**Pull request by bonzotronix**
* Instead of putting the JWT in a brkt-file with dest path /var/brkt/ami_config/token.jwt, put the JWT in the brkt_config dictionary with key 'identity_token'.

Testing:
* Ran unit tests
* Ran 'brkt make-user-data -jwt <file_with_jwt>'

### CODE REVIEW

File: brkt_cli/test_user_data.py [hunk 1]
```diff
@@ -58,12 +58,14 @@ def test_combine_user_data(self):
...
-                self.assertEqual('{"foo": "bar"}', content)
+                self.assertEqual(
+                    '{"foo": "bar", "brkt": {"identity_token": "%s"}}' % jwt,
+                    content)
             if part.get_content_type() == BRKT_FILES_CONTENT_TYPE:
                 found_brkt_files = True
                 content = part.get_payload(decode=True)
                 self.assertTrue('/var/brkt/ami_config/proxy.yaml:' in content)
-                self.assertTrue('/var/brkt/ami_config/token.jwt:' in content)
+                self.assertFalse('/var/brkt/ami_config/token.jwt:' in content)
```
Comment (bburtin) [inline brkt_cli/test_user_data.py]: I'd say remove this, since it's pretty much cruft now.

File: brkt_cli/user_data.py [hunk 2]
```diff
@@ -155,19 +157,24 @@ def to_mime_text(self):
         return str(container)
 
 
-def combine_user_data(brkt_config=None, proxy_config=None, jwt=None):
+def combine_user_data(brkt_config=None, proxy_config=None, jwt=None,
+                      do_gzip=True):
```
Comment (bburtin) [inline brkt_cli/user_data.py]: How hard would it be to break the gzip code out to a separate method in util and call it from the callsite?  That seems cleaner.

File: brkt_cli/test_user_data.py (hunk 1 above)
Comment (bonzotronix) [inline brkt_cli/test_user_data.py]: Done

File: brkt_cli/user_data.py (hunk 2 above)
Comment (bonzotronix) [inline brkt_cli/user_data.py]: Good idea.  Done

File: brkt_cli/user_data.py [hunk 3]
```diff
@@ -20,6 +20,8 @@
 
 import yaml
 
+from brkt_cli.util import add_token_to_user_data
```
Comment (bburtin) [inline brkt_cli/user_data.py]: While you're in this code, can you rename `add_token_to_user_data()` to `add_token_to_brkt_config()`?  I'm trying to be more consistent with differentiating the terminology.  Chances are that there's also a GCE callsite.

**Comment by bburtin** [comment]
Looks good, aside from the terminology nit.  Merge away once that's cleaned up.

File: brkt_cli/user_data.py (hunk 3 above)
Comment (bonzotronix) [inline brkt_cli/user_data.py]: Sure.  Done.

**Pull request merged**